src/app/globals.css

```diff
@@ -13,6 +13,7 @@
 
   --clr-yellow: #ffd966;
   --clr-darkblue: #122c3c;
+  --clr-darkblue-transparent: rgba(18, 44, 60, 0.5);
```

Comment:
These variable names are a bit too specific. They should describe their purpose and not the color they display. I recommend fixing that on a different branch after merging this